tools/protocols/src/LightValidator.ts

```diff
@@ -0,0 +1,72 @@
+import { ethers } from 'ethers'
+import { BigNumber } from 'ethers'
+import { chainIds, chainNames } from '@airswap/constants'
+import { LightOrder } from '@airswap/types'
+import * as LightValidatorContract from '@airswap/light-validator/build/contracts/LightValidator.sol/LightValidator.json'
+import * as LightValidatorDeploys from '@airswap/light/deploys.js'
```

Comment:
`@airswap/light-validator/deploys.js`

Reply:
with the rename to @airswap/validator, this should then be `@airswap/validator/deploys.js` no?

Reply:
Correct! Just wanted to be consistent with the current state.